src/SteelCap/Bootstrap/TextareaTagHelper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+using SteelCap.Extensions;
+
+namespace SteelCap.Bootstrap
+{
+    [TargetElement("sc-textarea")]
+    public class TextareaTagHelper : TagHelper
+    {
+        public override void Process(TagHelperContext context, TagHelperOutput output)
+        {
+            output.TagName = "textarea";
+
+            output.AppendClass("form-control");
```

Comment:
There's an extension method in Extensions/TagHelperExtensions.cs. to AppendClass.

Should we not use that one in all the tag helpers and remove the AppendClass from TextBoxCore, since they both do the same thing.